apps/vaults/contexts/useActionFlow.tsx

```diff
@@ -14,14 +14,27 @@ import {useYearn} from '@common/contexts/useYearn';
 
 import externalzapOutTokenList from '../../common/utils/externalZapOutTokenList.json';
 
+import type {BigNumber} from 'ethers';
 import type {ReactNode} from 'react';
+import type {TAddress} from '@yearn-finance/web-lib/utils/address';
 import type {TDropdownOption, TNormalizedBN} from '@common/types/types';
 import type {TYearnVault} from '@common/types/yearn';
 
+export enum	Flows {
```

Comment:
Should be singular, for enums
```suggestion
export enum	Flow {
```